pkg/issuer/venafi/client/request.go

```diff
@@ -158,6 +158,7 @@ func convertCustomFieldsToVcert(customFields []api.CustomField) ([]certificate.C
 
 func newVRequest(cert *x509.Certificate) *certificate.Request {
 	req := certificate.NewRequest(cert)
+	req.ChainOption = certificate.ChainOptionRootLast
```

Comment:
This forces TPP (and hopefully Cloud) to always put the root CA last in the list of PEM blocks.